packages/main/src/plugin/configuration-impl.ts

```diff
@@ -22,6 +22,14 @@ import { CONFIGURATION_DEFAULT_SCOPE } from '/@api/configuration/constants.js';
 
 import type { ApiSenderType } from './api.js';
 
+export interface ConfigurationChangedArg {
```

Comment:
Can't we reuse IConfigurationChangeEvent ? would be compliant with our extensions get notified

https://github.com/podman-desktop/podman-desktop/blob/80973f33fc6ea17bcf3be847b69b3b836f74cf5f/packages/main/src/plugin/configuration-registry.ts#L45-L49